yarn.lock

```diff
@@ -3906,10 +3918,9 @@ event-emitter@~0.3.5:
     d "1"
     es5-ext "~0.10.14"
 
-event-stream@3.3.4, event-stream@^3.3.4:
+event-stream@3.3.4, event-stream@^4.0.1:
```

Comment:
@markdboyd So I don't know the format of the lockfile.  I'm pretty sure this is saying we've installed event-stream@3.3.4, in place of the requested 4.0.1, but I don't know how you updated the lockfile before in your #517 PR.

Also, I got warnings for `warning Resolution field "event-stream@3.3.4" is incompatible with requested version "event-stream@^4.0.1"` 

But the tests pass.  Do we need to do something else?

Reply:
5d2093e254eed0c8159ab2d5d106a0f84c1813cc